code/game/objects/items/weapons/twohanded.dm

```diff
@@ -336,7 +336,18 @@
 		if(input)
 			src.war_cry = input
 
-/obj/item/weapon/twohanded/spear/CheckParts()
+//Placeholder C4 "grenade" for use on this spear
+/obj/item/weapon/grenade/C4
+	name = "C-4"
+	desc = "A brick of C-4."
+
+/obj/item/weapon/grenade/C4/prime()
+	update_mob()
+	explosion(src.loc,-1,1,3)
+	qdel(src)
+
```

Comment:
i assumed as much. You might have messed up the chembomb recipe again then as well.